### ISSUE TITLE
[Android] Cannot build extension because of gradle 8.7

### ISSUE THREAD

**Issue by shichen85** [YoYoGames/GameMaker-Bugs#7973]
### Description
The sample project can build in the April runtime after the fix of https://github.com/YoYoGames/GameMaker-Bugs/issues/5173, but fails to build with the latest runtime because gradle was upgraded from 7.4.0 to 8.5.0. The following was thrown:

```
* What went wrong:
A problem occurred configuring project ':library'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file: C:\temp\GMS2CACHE\GooglePlay_71A54692_4B59922\android\Default\library\build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.
```

This can be mediated if the runtime's `android/runner/ExtensionFiles/build.gradle` also contains `namespace '${YYAndroidPackageName}'` as `android/runner/ProjectFiles/build.gradle` does, which is the solution we recommend. 

See https://stackoverflow.com/questions/78744324/could-not-create-an-instance-of-type-com-android-build-api-variant-impl-libraryv for reference.

### Steps To Reproduce
1. Build the project for Android VM with Nov beta runtime. See error
2. Edit `android/runner/ExtensionFiles/build.gradle` to also contains `namespace '${YYAndroidPackageName}'`.
3. Build again and succeed

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.618 Runtime v2024.1100.0.664

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22000.0

### Which platform(s) are you seeing the problem on?
Android

[2dda0e16-2fd7-4e7c-aa8e-ea52e39b92bd](https://api.gamemaker.io/api/github/downloads/2dda0e16-2fd7-4e7c-aa8e-ea52e39b92bd)

**Comment by yoyo-danielc** [comment]
Suggested fix integrated to 2024.11 branch, thank you

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.1100.0.653  Runtime v2024.1100.0.679
